### ISSUE TITLE
Might want to add a note about bumping the package version as well (see my comment in [this PR](https://github.com/pharmaverse/admiraltemplate/pull/72#pullrequestreview-1872726088))

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiralpeds#3]
Might want to add a note about bumping the package version as well (see my comment in [this PR](https://github.com/pharmaverse/admiraltemplate/pull/72#pullrequestreview-1872726088))

_Originally posted by @manciniedoardo in https://github.com/pharmaverse/admiralpeds/pull/1#discussion_r1484517383_

**Comment by rossfarrugia** [comment]
@bms63 @zdz2101 do we only have to add this after we release our first version?

**Comment by manciniedoardo** [comment]
I'll look into this

**Comment by rossfarrugia** [comment]
@manciniedoardo i was thinking about this and we only actually need to implement this after our first foundational 0.1.0 release, as nobody is really going to install latest github version until then. we don't even need to update NEWS file until then either, as will explain now on slack channel

**Comment by manciniedoardo** [comment]
Makes sense - I’ve since located the documentation for how to implement this, so if you change your mind I can set this up. Otherwise we can keep as an action item for after first release.

**Comment by rossfarrugia** [comment]
@manciniedoardo feel free to add any PR for this now as needed, as we're on CRAN. The action does fire already for us though and me and @Fanny-Gautier learned that we should use [skip vbump] in PR title if ever we want to bipass this.

**Comment by rossfarrugia** [comment]
@manciniedoardo is anything needed updating here or could we close this one out? me and @Fanny-Gautier are looking at wrapping up admiralpeds now ready for the Jan release. tbh we think this package will be relatively stable from here on out as we've not heard much user feedback on anything to change or any new features needed.
